File: atsa/src/main/java/jp/kaiz/atsassistmod/controller/trainprotection/ATSPsController.java
// SPDX-License-Identifier: LGPL-3.0-or-later
// Copyright © 2026 mirukuneko and RealTrainModRenewed contributors
package jp.kaiz.atsassistmod.controller.trainprotection;

public class ATSPsController extends TrainProtection {

    @Override
    public TrainProtectionType getType() {
        return TrainProtectionType.ATSPs;
    }
}

File: atsa/src/main/kotlin/jp/kaiz/atsassistmod/controller/trainprotection/ATSPsController.kt
// SPDX-License-Identifier: LGPL-3.0-or-later
// Copyright © 2026 mirukuneko and RealTrainModRenewed contributors
package jp.kaiz.atsassistmod.controller.trainprotection

open class ATSPsController : TrainProtection() {
    override fun getType(): TrainProtectionType = TrainProtectionType.ATSPs
}
